modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -439,6 +439,23 @@ private function add_content_style_section() {
 			[
 				'name' => 'content_border',
 				'selector' => $low_specificity_accordion_item_selector,
+				'fields_options' => [
+					'color' => [
+						'default' => '#D5D8DC',
+					],
+					'width' => [
+						'default' => [
+							'top' => 1,
+							'right' => 1,
+							'bottom' => 1,
+							'left' => 1,
+							'unit' => 'px',
+						],
+					],
+					'border' => [
+						'default' => 'solid',
+					],
```

Comment:
We usually set the style defaults in the CSS file itself.
But I guess this is valid as well.